.github/workflows/stale-bot.yml

```diff
@@ -0,0 +1,21 @@
+name: 'Label stale issues and PRs'
+on:
+  schedule:
+    - cron: '30 1 * * 1' # runs once a week
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/stale@v4
+        with:
+          stale-issue-message: '' # no comment left if string is empty
+          stale-pr-message: '' # no comment left if string is empty
+          days-before-stale: 30
+          days-before-close: -1
+          stale-issue-label: 'needs attention'
+          stale-pr-label: 'needs attention'
+          exempt-issue-labels: 'good intro to dask,good first issue,Good First Issue,good second issue,feature request'
+          #exempt-pr-labels: 'awaiting-approval,work-in-progress'
+          exempt-draft-pr: true
+          start-date: '2020-04-18T00:00:00Z' # ignore before this date, ISO 8601 or RFC 2822
```

Comment:
Why would we want to ignore tickets before this date? @jsignell do you know why this is in here?

Reply:
I guess the idea was not to apply them to all of history, but only going forward. Based on the conversation in the dev meeting, seems like we are ok with letting the bot do more.